### ISSUE TITLE
Support for top level "parameters".

### ISSUE THREAD

**Issue by shindex-rhems** [suzuki-shunsuke/circleci-config-merge#35]
Nice to meet you.
I had the same problem as you and found your blog. Since then, I have been using `circleci-config-merge`.

I have one request.

We use [conditional workflows](https://circleci.com/docs/2.0/pipeline-variables/#conditional-workflows) in some cases.
However, `circleci-config-merge` does not merge the top-level parameters properly.

```
version: 2.1

parameters: // THIS IS NOT MERGED.
  run_integration_tests:
    type: boolean
    default: false

workflows:
  version: 2
  integration_tests:
    when: << pipeline.parameters.run_integration_tests >>
    jobs:
      - mytestjob

jobs:
  mytestjob:
    steps:
      - checkout
      - ... # job steps
```

Is it possible to fix this problem with an update?

**Comment by suzuki-shunsuke** [comment]
Thank you for your request.
I have created https://github.com/suzuki-shunsuke/circleci-config-merge/pull/41

**Comment by shindex-rhems** [comment]
It's beautiful !!!!
Thank you very much for your quick response.

**Comment by suzuki-shunsuke** [comment]
I have released [v1.1.0](https://github.com/suzuki-shunsuke/circleci-config-merge/releases/tag/v1.1.0).
